src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -107,14 +117,75 @@ export class ExamUpdateComponent implements OnInit {
         return this.exam?.maxPoints !== undefined && this.exam?.maxPoints > 0;
     }
 
+    /**
+     * Validates the given StartDate.
+     * For RealExams, the visibleDate has to be strictly prior the startDate.
+     * For TestExams, the visible Date has to be prior or equal to the startDate.
```

Comment:
Why do we need this distinction?

Reply:
For RealExams the visible date must be strictly prior to the start date, so students can open the start screen, agree on the terms & conditions, and wait for the RealExam to start. As the working time is the difference between start date and end date, not having the visible date strictly prior to the start date would cause the students to miss working time.

For TestExams, the decision was made to allow the visible date to be equal to the start date. As the working time now depends on value working time, students have exactly the working time available, no matter when the TestExams is started by the student (constraint by the end of the working window). 
Furthermore, instructors can set the visible date equal to the start date to enable students to conduct the TestExam right after the TestExam is available to the students *(similar to exercises on Artemis, e.g. if the TestExam is provided for practice)*. But instructors can define a prior visible date to mock the conditions of the RealExam *(e.g. to familiarize the students with the modalities of the RealExam Mode.)*